packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -157,6 +157,7 @@ export async function queryAndProcessDocuments({
         forceDownload === undefined &&
         ignoreDocRefOnFHIRServer === undefined
       ) {
+        // THIS WILL NEED TO BE MOVED TO A DIFFERENT PLACE WHEN WE SUPPORT MULTIPLE HIEs
         reportDocQueryUsage(patient);
       }
```

Comment:
Im wondering if this could be moved to the file that calls both HIE's to initialize doc query? If not maybe when downloading or converting is complete maybe we mark that usage has been made

Reply:
Agree, maybe when we identify we should send the WH notification? I guess it depends on the definition of what is a doc query usage.